design/ceph/object/store.md

```diff
@@ -103,6 +104,14 @@ The RGW service can be configured to listen on both http and https by specifying
     instances: 1
 ```
 
+#### Hostname Configuration
+
+These hostnames will be added to the zonegroup's hostname list and objectstore can be accessed only by the hostname from the list. If the feature is enabled, users need to add the all the endpoints like the openshift route, ingress loadbalancer and k8s service endpoint created by Rook to the list. The hostname list can be updated at any time by updating the object store CRD. Even though settings related to `zonegroup` to make consistent for single and multisite use case it is added to the object store CRD. After creating the object store, the Rook operator will update the zonegroup to include the `hostnames` to its list via `radosgw-admin` command.
```

Comment:
> the Rook operator will update the zonegroup to include the `hostnames` to its list via `radosgw-admin` command.

Please also show an example command of how the operator can add the hostname to the list.